src_new/creative/owCreativeRenderer.js

```diff
@@ -0,0 +1,44 @@
+require("prebid-universal-creative");
+
+window.PWT = window.PWT || {};
+window.pbjs = window.pbjs || {};
+
+var refThis = this;
+
+// Desc: Uses prebid js render ad function to call cache and render ad based on the params from response.
+// Param Defination 
+exports.renderCreative = function (cacheUrl, cachePath, uuid) {
```

Comment:
we had discussed to pass document as a parameter and all PubMatic specific params in an object as another parameter.